### ISSUE TITLE
Disable emoji and colors in logs by default in windows 

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#5018]
I keep forgetting to open this but the Unix shell color codes and emoji in the logs don't work in PowerShell or Cmd.exe so we should just default to disabling those on windows

**Comment by salsakran** [comment]
This is pretty embarrassing, even if it isn't harmful to functionality.

**Comment by camsaul** [comment]
It's not a big deal they just pop up as question marks.

**Comment by salsakran** [comment]
That doesn't exactly inspire trust when someone is first trying us out.

**Comment by camsaul** [comment]
Fixed by #5037

**Comment by camsaul** [comment]
Although Metabase is correctly detecting the OS as Windows this doesn't seem to be disabling the ANSI color escape codes as expected. It looks like there might be some other issue with configuring these settings. Manually configuring the env vars doesn't seem to work *either* for log colorization, but emoji do seem to be getting disabled as expected.

```powershell
$env:MB_COLORIZE_LOGS = 'false'
$env:MB_EMOJI_IN_LOGS = 'false'
java -jar metabase.jar
```

**Comment by Varunram** [comment]
@camsaul Exporting the vars in UNIX does disable the emoji and colour. Maybe some weird Windows issue?

**Comment by camsaul** [comment]
> the Unix shell color codes and emoji in the logs don't work in PowerShell or Cmd.exe

Yes @Varunram, that's what I said earlier 😉

**Comment by Varunram** [comment]
A problem with my comprehension :) Windows never ceases to amaze me though

**Comment by camsaul** [comment]
Fixed by #5899
